src/D2L.Bmx/Okta/OktaSessionStorage.cs

```diff
@@ -6,43 +6,36 @@ namespace D2L.Bmx.Okta;
 
 internal interface IOktaSessionStorage {
 	void SaveSessions( List<OktaSessionCache> sessions );
-	List<OktaSessionCache> Sessions();
+	List<OktaSessionCache> GetSessions();
 }
 
 internal class OktaSessionStorage : IOktaSessionStorage {
 	void IOktaSessionStorage.SaveSessions( List<OktaSessionCache> sessions ) {
+		if( !Directory.Exists( BmxPaths.BMX_DIR ) ) {
+			Directory.CreateDirectory( BmxPaths.BMX_DIR );
+		}
```

Comment:
It worked fine without this because we always read the session file before writing to it.
But
* We don't create the file on read now
* This method should be safe to call regardless of whether we called the read method before